lib/BatchMessageAcker.h

```diff
@@ -37,14 +37,16 @@ class BatchMessageAcker {
     // by deserializing from raw bytes.
```

Comment:
The modification of this class to have solved the compilation error. Can we first open a new PR to merge it?

Reply:
I will modify this PR after https://github.com/apache/pulsar-client-cpp/pull/177 is merged.